cardano-node/src/Cardano/Node/Protocol/Shelley.hs

```diff
@@ -163,14 +164,19 @@ readLeaderCredentials (Just ProtocolFilepaths {
     KesSigningKey kesKey <-
       firstExceptT FileError . newExceptT $ readFileTextEnvelope (AsSigningKey AsKesKey) kesFile
 
+    -- The label is used to distinguish multiple sets of credentials during
+    -- tracing. We use the public KES key as the label.
+    let label = T.pack (show (deriveVerKeyKES kesKey))
```

Comment:
Another reason for replacing this with just `"Shelley"`, is that we have to be careful not to hold on to the `kesKey` itself. A bang on `label` here would be enough, though.

Reply:
I cannot find this in the PR anymore, maybe you resolved this in some other way.

Reply:
Yeah, this has been resolved already, it was just a comment pointing out the bug in the original code (missing bang).